src/erlcloud_cloudformation.erl

```diff
@@ -405,7 +404,8 @@ convert_param({Key, Value}) ->
         stack_name -> {"StackName", Value};
         stack_id -> {"StackName", Value};
         logical_resource_id -> {"LogicalResourceId", Value};
-        next_token -> {"NextToken", Value}
+        next_token -> {"NextToken", Value};
+        _ -> {Key, Value}
```

Comment:
That's what erlcloud should do ideally - support param names only in the same format they are written in the API. But it seems like this module has support for atoms as param names in a few functions, so it might make sense to implement it for the sake of module consistency. As long as `{"", ""}` is still fully supported, I think it's fine